### ISSUE TITLE
Load Data parsing failure

### ISSUE THREAD

**Issue by joelvbernier** [HEXRD/hexrdgui#281]
The following filenames give a load error:

```
mruby-0129_0004-ff1_0_0.npz
mruby-0129_0004-ff1_0_1.npz
mruby-0129_0004-ff1_1_0.npz
mruby-0129_0004-ff1_1_1.npz
mruby-0129_0004-ff2_0_0.npz
mruby-0129_0004-ff2_0_1.npz
mruby-0129_0004-ff2_1_0.npz
mruby-0129_0004-ff2_1_1.npz
```
for the detector keys:
```
ff1_0_0
ff1_0_1
...
ff2_1_1
``` 
This is for the current master.

**Comment by joelvbernier** [comment]
Looks like we should in general just be able to use the `find()` builtin method for strings, no?

```
imsname = 'mruby-0129_0004.ff1_0_1.npz'

imsname.find('ff1_0_1')
Out[102]: 16

imsname.find('ff2_0_1')
Out[103]: -1
```

**Comment by bnmajor** [comment]
Closing for now as this seems to be fixed with #282, we can always re-open if there are any issues.
